examples/admin_create_acls/admin_create_acls.go

```diff
@@ -0,0 +1,142 @@
+		principal := args[start+3]
+		host := args[start+4]
+		operationString := args[start+5]
+		permissionTypeString := args[start+6]
+
+		var resourceType kafka.ResourceType
+		var resourcePatternType kafka.ResourcePatternType
+		var operation kafka.AclOperation
+		var permissionType kafka.AclPermissionType
+
+		resourceType, err = kafka.ResourceTypeFromString(resourceTypeString)
+		if err != nil {
+			fmt.Printf("Invalid resource type: %s: %v\n", resourceTypeString, err)
+			return
+		}
+		resourcePatternType, err = kafka.ResourcePatternTypeFromString(resourcePatternTypeString)
+		if err != nil {
+			fmt.Printf("Invalid resource pattern type: %s: %v\n", resourcePatternTypeString, err)
+			return
+		}
+
+		operation, err = kafka.AclOperationFromString(operationString)
+		if err != nil {
+			fmt.Printf("Invalid operation: %s: %v\n", operationString, err)
+			return
+		}
+
+		permissionType, err = kafka.AclPermissionTypeFromString(permissionTypeString)
+		if err != nil {
+			fmt.Printf("Invalid permission type: %s: %v\n", permissionTypeString, err)
+			return
+		}
+
+		parsedAclBindings[i] = kafka.AclBinding{
+			Type:                resourceType,
+			Name:                name,
+			ResourcePatternType: resourcePatternType,
+			Principal:           principal,
+			Host:                host,
+			Operation:           operation,
+			PermissionType:      permissionType,
+		}
+	}
+	aclBindings = parsedAclBindings
+	return
+}
+
+func main() {
+
+	// 2 + 7n arguments to create n ACL bindings
+	nArgs := len(os.Args)
+	aclBindingArgs := nArgs - 2
+	if aclBindingArgs <= 0 || aclBindingArgs%7 != 0 {
+		fmt.Fprintf(os.Stderr,
+			"Usage: %s <broker> <resource-type1> <resource-name1> <resource-pattern-type1> "+
+				"<principal1> <host1> <operation1> <permission-type1> ...\n",
+			os.Args[0])
+		os.Exit(1)
+	}
+
+	broker := os.Args[1]
+	aclBindings, err := parseAclBindings(os.Args[2:])
+	if err != nil {
+		os.Exit(1)
+	}
+
+	// Create a new AdminClient.
+	// AdminClient can also be instantiated using an existing
+	// Producer or Consumer instance, see NewAdminClientFromProducer and
+	// NewAdminClientFromConsumer.
+	a, err := kafka.NewAdminClient(&kafka.ConfigMap{"bootstrap.servers": broker})
+	if err != nil {
+		fmt.Printf("Failed to create Admin client: %s\n", err)
+		os.Exit(1)
+	}
+
+	// Contexts are used to abort or limit the amount of time
+	// the Admin call blocks waiting for a result.
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Create ACLs on cluster.
+	// Set Admin options to wait for the request to finish (or at most 60s)
+	maxDur, err := time.ParseDuration("60s")
+	if err != nil {
+		panic("ParseDuration(60s)")
+	}
+	results, err := a.CreateAcls(
+		ctx,
+		aclBindings,
+		kafka.SetAdminRequestTimeout(maxDur),
+	)
+	if err != nil {
+		fmt.Printf("Failed to create ACLs: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Print results
+	for _, result := range results {
+		fmt.Printf("%+v\n", result)
```

Comment:
If there's per-ACL error reporting I think it would be great to show-case that here since otherwise people will assume that the operation is always successful at this point.